src/components/access-type-select/access-type-select-field.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'react-final-form';
+import { FormattedMessage } from 'react-intl';
+
+import { Select } from '@folio/stripes/components';
+
+import { accessTypes } from '../../constants';
+
+const propTypes = {
+  accessStatusTypes: PropTypes.object.isRequired,
+};
+
+function AccessTypeSelectField({ accessStatusTypes }) {
+  const accessTypesRecords = accessStatusTypes.resolver.state.accessTypes.records;
```

Comment:
the same about passing formatted data. before passing `accessStatusTypes` to `AccessTypeSelectField`, I'd format it like this: 
```js
[
  {name: 'name', id: 'id'},
  {name: 'name', id: 'id'},
  {name: 'name', id: 'id'}, 
]
```